_events/2015-08-29.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Music Games Hackathon"
+description: "Music Games Hackathon"
```

Comment:
I know these are old events so it doesn't matter much but the description doesn't add much. On the Past Events page, it will just say Music Education Hackathon twice which isn't great...

<img width="440" alt="screen shot 2017-04-24 at 08 58 03" src="https://cloud.githubusercontent.com/assets/891252/25338171/48e4dbe6-28cc-11e7-82bb-ea574a652913.png">